crates/aiken-lang/src/tipo/error.rs

```diff
@@ -1076,6 +1076,21 @@ pub enum Warning {
         sample: UntypedExpr,
     },
 
+    #[error("I found an {} trying to match a type with one constructor", "expect".purple())]
+    #[diagnostic(
+        code("single_constructor_expect"),
+        help("If your type has one constructor, unless you are casting {} {}, you can\nprefer using a {} binding like so...\n\n{}", "FROM".bold(), "Data".purple(), "let".purple(), format_suggestion(sample))
+    )]
+    SingleConstructorExpect {
+        #[label("use let")]
+        location: Span,
+        #[label("only one constructor")]
+        pattern_location: Span,
+        #[label("is not {}", "Data".purple())]
```

Comment:
```suggestion
        #[label("is not Data")]
```

I'd avoid the coloring here because labels are already colored. Given that we've already got 2 others labels, that's 4 colors on one line 😅